client/src/pages/DuelOverview.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import axios from "axios";
+import AuthService from "../services/auth.service";
+
+import Avatar from "../components/Avatar";
+
+const UserBadge = ({ user, reversed }) => {
```

Comment:
Maybe you could add the proptypes? Now that we're using the right package ^^

Reply:
You are right!